bundles/asdi/asdi-projection-change/instance.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @class Oskari.projection.change.instance
+ * 
+ *      var obj = {
+            "bundlename":"asdi-projection-change" ,
+            "metadata": {
+                "Import-Bundle": { "asdi-projection-change": { "bundlePath": "/Oskari/packages/asdi/bundle/" } }
+            }
+        }
+        appSetup.startupSequence.push(obj);
+ */
+Oskari.clazz.define("Oskari.projection.change.instance",
+function () {
+
+}, {
+    __name: 'projection-change',
+    getName: function () {
+        return this.__name;
+    },
+    afterStart: function () {
+        var sandbox = this.getSandbox();
+        this._mapmodule = sandbox.findRegisteredModuleInstance('MainMapModule');
+        this.plugins['Oskari.userinterface.Flyout'] = Oskari.clazz.create('Oskari.projection.change.flyout', this, {
```

Comment:
Why would you extend DefaultExtension AND override the plugins here? I'd say https://github.com/oskariorg/oskari-frontend/blob/master/src/BasicBundle.js feels more likely candidate for what you should be extending.